### PR TITLE
INFRADEV-12144: Allow sorting file list

### PR DESCRIPTION
**Pull request by DorAmram**

### CODE REVIEW

File: webapp/app/components/beam-files/component.js [hunk 1]
```diff
@@ -62,6 +65,11 @@ export default Component.extend({
   actions: {
     filterChange: function(newFilter) {
       this.set("fileFilter", newFilter);
+    },
+    sortBy: function(property, acsending) {
+      var method = (acsending) ? 'asc': 'desc';
```
Comment (r-darwish) [inline webapp/app/components/beam-files/component.js]: Better use `const` or `let`

https://medium.com/@pandeysoni/when-should-use-const-and-let-instead-of-var-in-javascript-ec2c3d7e5ca6

File: webapp/app/components/beam-files/component.js [hunk 2]
```diff
@@ -62,6 +65,11 @@ export default Component.extend({
...
+      this.get('sortKeys').pop()
```
Comment (r-darwish) [inline webapp/app/components/beam-files/component.js]: I think that you don't have to use `this.get` in recent versions of Ember. See if `this.sortKeys.pop()` works.

File: webapp/app/components/sortable-header/component.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+    downArrow: '&#9660',
```
Comment (r-darwish) [inline webapp/app/components/sortable-header/component.js]: You don't need to put these as variables. Put them directly inside the template.

Also, consider font awesome icons
https://fontawesome.com/icons?d=gallery&q=down

File: webapp/app/components/sortable-header/component.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    upArrow: '&#9650',
+
+    tagName: 'th',  
+
+    actions: {
+        sortBy: function(sortProperty, acsending){
+            this.get('sortBy')(sortProperty, acsending);
```
Comment (r-darwish) [inline webapp/app/components/sortable-header/component.js]: I think that you can do `this.sortBy(sortProperties, acsending)`

File: webapp/app/components/sortable-header/component.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+    downArrow: '&#9660',
+    upArrow: '&#9650',
+
+    tagName: 'th',  
+
+    actions: {
+        sortBy: function(sortProperty, acsending){
```
Comment (r-darwish) [inline webapp/app/components/sortable-header/component.js]: I prefer `sortBy(sortProperty, acsending) {`. No need to use `function`

File: webapp/app/components/sortable-header/component.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import Component from '@ember/component';
```
Comment (r-darwish) [inline webapp/app/components/sortable-header/component.js:1]: Good use of a component 👍

**Review by r-darwish — CHANGES_REQUESTED** [review]

File: webapp/app/components/sortable-header/component.js (hunk 6 above)
Comment (DorAmram) [reply webapp/app/components/sortable-header/component.js:1]: Thanks :)

**Review by r-darwish — APPROVED** [review]

**Pull request merged**